tests/testthat/test-util-checkSpec.R

```diff
@@ -159,3 +159,36 @@ test_that("Validate column type works", {
     regexp = "Not all columns"
   )
 })
+
+test_that("skip column check when `_all` is specified", {
+  # example lSpec
+  lSpec <- list(
+    df1 = list(
+      a = list(required = TRUE),
+      b = list(required = TRUE)
+    ),
+    df2 = list(
+      x = list(required = TRUE),
+      y = list(required = TRUE)
+    ),
+    df3 = list(
+      `_all` = list(required = TRUE)
+      )
+  )
+
+  # Example data
+  lData <- list(
+    df1 = data.frame(a = 1:3, b = 4:6),
+    df2 = data.frame(x = 7:9, y = 10:12),
+    df3 = data.frame(z = 1:3, t = 20:22)
+  )
+
+  expect_message(
+    expect_message(expect_message(
+      expect_message(CheckSpec(lData, lSpec), "All 3 data"),
+      "All specified"
+    ), "All specified"),
+    "All 4 required"
```

Comment:
4 includes `a`, `b`, `x`, and `y`?

Reply:
right. at this point, we are only checking columns that specify particular columns, not those that require `_all`. I suppose, I could add those columns to the message count, but it might complicate things a bit.